web/src/app/components/resources/list/list.component.ts

```diff
@@ -25,33 +30,45 @@ export class ListComponent implements AfterViewInit {
   ];
   @ViewChild(MatPaginator) paginator: MatPaginator;
   @ViewChild(MatSort) sort: MatSort;
-  filterEmitter = new EventEmitter();
 
-  resources: EsHits<Resource>[] = [];
+  resources: Observable<EsHits<Resource>[]>;
   total = 0;
   isLoadingResults = true;
-  private query: string;
+  form$: Observable<FormGroupState<SearchParams>>;
+  private controls: FormGroupControls<SearchParams>;
+  private formSubscription: Subscription;
 
-  constructor(private resourceService: ResourceService) {
+  constructor(private resourceService: ResourceService, private store: Store<State>) {
   }
 
-  ngAfterViewInit(): void {
+  ngOnInit(): void {
+    this.form$ = this.store.select('searchForm');
+  }
+
+  ngOnDestroy(): void {
+    this.formSubscription?.unsubscribe();
+  }
 
-    merge(this.paginator.page, this.sort.sortChange, this.filterEmitter)
+  ngAfterViewInit(): void {
+    this.formSubscription = this.form$.subscribe(form => this.controls = form.controls);
+    this.resources = merge(this.paginator.page, this.sort.sortChange, this.form$)
       .pipe(
         startWith({}),
-        switchMap(() => {
+        switchMap((value) => {
           this.isLoadingResults = true;
```

Comment:
You can use `tap` operator for `this.isLoadingResults`.

Reply:
done